docs/spec/pagination/pagination.examples.boundary-links.use-case.md

```diff
@@ -0,0 +1,26 @@
+8.5: Pagination boundary links examples
+=======================================
+
+ **Primary Actor**: User 
+ 
+ **Scope**: Ngx-bootstrap DEMO / BS version 3&4
+ 
+ **Goal**: Show user boundary links functionality
+ 
+ Main success scenario:
+----------------------
+
+ 1. User opens Pagination demo page
+ 2. User clicks on Boundary links event sub-menu
+ 3. By default 'First' and 'Previous' buttons are disabled, and 1st page is active
+ 4. After click on 'Last' button, 6th page is active, 1st page is inactive, 'Next' and 'Last' buttons are disabled,
```

Comment:
quotes should be the same in each use-case. Please, apply this rule to all use-cases